econplayground/templates/main/graph_form.html

```diff
@@ -10,6 +10,10 @@
 {% endblock %}
 
 {% block js %}
+    if (!window.EconPlayground) {
+        window.EconPlayground = {};
+    }
+    window.Econplayground[graphType] =  {{graph_type}}
```

Comment:
This block of code needs to be surrounded by a `<script>` tag:
```
<script>
    if (!window.EconPlayground) {
        window.EconPlayground = {};
    }
    window.EconPlayground[graphType] =  {{graph_type}};   // <-- also add semicolon to this line
    // and fix typo in EconPlayground variable here, capitalizing the P.
</script>
```

Reply:
Okay will do. Why is the js block not already in a script tag?